server/views/pages/yourCompletedReports.njk

```diff
@@ -46,18 +45,17 @@
 
     {{ govukTable({
       head: [
-        { html: 'Prisoner&#8217;s name' },
-        { text: "Prison number" },
-        { text: "Date and time of incident",
+        { text: "Discovery date and time",
           attributes: {
           "aria-sort": "descending"
         }
         },
+        { html: 'Name and prison number' },
```

Comment:
I think this can just be 'text' rather than html
`{ text: 'Name and prison number' },`